evaluator/evaluator_test.go

```diff
@@ -1101,7 +1108,10 @@ func TestCommand(t *testing.T) {
 		{"`echo hello world | xargs echo -n`", "hello world"},
 		{"`echo \\$CONTEXT`", "abs"},
 	}
-
+	if runtime.GOOS == "windows" {
```

Comment:
here it's different instead. Since the majority of these are unix commands I think it's ok to keep these out of the windows test suite for now, so :+1: I just have to remember that when we merge this I should open an issue to make sure we have tests for win commands as well